src/SFA.DAS.LearnerDataMismatches.Web/Infrastructure/ProviderService.cs

```diff
@@ -22,7 +22,7 @@ public string GetProviderName(long ukprn)
                 var provider = _providerApiClient.Get(ukprn);
                 return provider.ProviderName;
             }
-            catch (HttpRequestException)
+            catch
```

Comment:
This was deliberately kept to set the expectation that we are expecting this error in case the ukprn is wrong.

Reply:
I think that any error getting the names should not necessarily prevent the report page from loading.

I saw the entire report show an exception message because the provider API was failing.